docs/source/getting-started/model-registry/step1-register-model.rst

```diff
@@ -0,0 +1,73 @@
+Register a Model
+=================
+
+To effectively use the Model Registry, it's necessary to first install and configure MLflow. In this
+quick start guide, we'll use a local tracking server which saves information about your MLflow 
+resources in a local directory. More specifically, models will be located in the `./mlruns/models` 
+directory.
+
+.. note::
+    For production use cases, we advise using a remote server. In this guide, we're using a local 
+    server for demonstration purposes.
+
+Step 1: Install MLflow from PyPI
+--------------------------------
+
+MLflow is conveniently available on PyPI. Installing it is as simple as running a pip command.
+
+.. code-section::
+    .. code-block:: bash
+        :name: download-mlflow
+
+        pip install mlflow 
+
```

Comment:
We can probably omit this step. You could start the next section with a link to the instructions on downloading MLflow from pypi (the first initial guide), or just omit it since this is not going to be tutorial entry point for MLflow, but rather a component quickstart.